src/components/ChatGhostUI/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {Dimensions} from 'react-native';
+import _ from 'underscore';
+import ContentLoadingOneLine from './ContentLoadingElements/ContentLoadingOneLine';
+import ContentLoadingTwoLines from './ContentLoadingElements/ContentLoadingTwoLines';
+import ContentLoadingThreeLines from './ContentLoadingElements/ContentLoadingThreeLines';
+
+
+const ChatList = () => {
+    const windowHeight = Dimensions.get('window').height;
+
+    const height = 80;
+    const numberOfRows = Math.floor(windowHeight / height);
+    const contentItems = Array.from({length: numberOfRows}, (v, i) => i);
+
+    return (
+        <>
+            {
+                _.map(contentItems, (item, index) => {
```

Comment:
```suggestion
                _.keys(contentItems, (index) => {
```

Reply:
I am guessing you mean `_.map(_.keys(contentItems))` ?

Reply:
Not specifically. My suggestion is to use something that only gives you indexes as the first parameter. If there is nothing like that, then no need to change this.

Reply:
Yeah thought about it, `_.keys` will only return an array, will still have to map it.